src/components/pages/home.js

```diff
@@ -0,0 +1,30 @@
+import { useSelector } from 'react-redux';
+import { useState } from 'react';
+import CaseComponent from '../caseComponent';
+
+const HomePage = () => {
+  const { all } = useSelector((state) => state.cases);
+  const [state] = useState({
+    value: '',
+    filteredCases: [],
+  });
+  return (
+    <>
```

Comment:
- [ ] Please make sure that it should be two cards in a row as in the design to follow the design guidelines.

### Your design
![image](https://user-images.githubusercontent.com/83803414/195617922-bd31b384-18a6-44a1-bc48-a4e01ee94f7c.png)

### Behance design
![image](https://user-images.githubusercontent.com/83803414/195618151-ed172115-65fb-40e9-9f5c-62c82b2cb4c9.png)